graph/origin/subgraph.yaml.mustache

```diff
@@ -0,0 +1,47 @@
+specVersion: 0.0.1
+schema:
+  file: ./schema.graphql
+dataSources:
+  - kind: ethereum/contract
+    name: OSTComposer
+    network: mainnet
```

Comment:
Why do we have "mainnet" as a network? I've checked subgraph spec documentation, this seems to be mandatory field, however, I am not sure for what it is used in our case (local setup).

Reply:
Yea, it's mandatory. I am also not sure about it's significant. From `graph init` command it gives few options like `Mainnet, Robsten, Koven`. I propose we can keep this as it is for now and as we use it we will understand it's significance.

Reply:
Agree.